src/services/bookService.js

```diff
@@ -13,6 +13,26 @@ class BookService {
     }
     return book;
   }
+
+  async getBooksByQueryType(queryType, page = 1, pageSize = 20) {
+    if (!queryType) {
+      throw new Error('Invalid query type');
+    }
+
+    page = Number.isInteger(parseInt(page)) && parseInt(page) > 0 ? parseInt(page) : 1;
+    pageSize = Number.isInteger(parseInt(pageSize)) && parseInt(pageSize) > 0 ? parseInt(pageSize) : 20;
```

Comment:
shouldn't it be declared `const pageSize` ?

Reply:
In a function declaration, default values initialize the parameters, so the variables cannot be redeclared with const or let inside the code block.